scripts/parks/ParkProvider.js

```diff
@@ -27,6 +27,17 @@ export const getParks = () => {
     })
 }
 
+export const getParkCoordinates = parkId => {
```

Comment:
Also, I think we should call getParkCoordinates useParkCoordiates, since it doesn't include a fetch call.